docs/index.html

```diff
@@ -189,6 +189,7 @@ <h2 id="components">Components</h2>
 </aside>
 
 
+<b>Note that the codes in this component section are meant to be examples, and not necessarily 1:1 reference to be used</b>
```

Comment:
use of `strong` is preferable to `b` tags here. I would suggest they be around the word Note only. Something like this. 

`````
<strong>Note:</strong> the codes in this component section are meant to be examples, and not necessarily 1:1 reference to be used exactly.</b>

Reply:
Okay 
I'll work on it and resend a pull request

Reply:
Hello @possumbilities kindly review the change I made
I have changed the code as requested